### ISSUE TITLE
jupyter_nbextensions_configurator is not compatible with notebook 7.0.0

### ISSUE THREAD

**Issue by alaabhs** [Jupyter-contrib/jupyter_nbextensions_configurator#165]
I have this exception when i run `jupyter nbextensions_configurator enable --sys-prefix`

```
Traceback (most recent call last):
  File "/home/alaa/code/test_env/test/bin/jupyter-nbextensions_configurator", line 5, in <module>
    from jupyter_nbextensions_configurator.application import main
  File "/home/alaa/code/test_env/test/lib/python3.10/site-packages/jupyter_nbextensions_configurator/__init__.py", line 18, in <module>
    from notebook.base.handlers import APIHandler, IPythonHandler
ModuleNotFoundError: No module named 'notebook.base'
```

**Comment by mfouesneau** [comment]
Same issue.

**Comment by davidfraser** [comment]
Would it be possible to update the metadata for juypter-nbextensions-configurator to indicate that it currently requires notebook < 7.0?

**Comment by WesleyTheGeolien** [comment]
Running the latest `main` this still did not seem to be working.

It appears it can be fixed by calling the `nbclassic.shim_notebook` for `notebook>7` this should be addressed in #171
